Only in powiedz_0.2: Revision.o
Only in powiedz_0.2: aufile.o
diff -ur powiedz_0.2.orig/def_pars.c powiedz_0.2/def_pars.c
--- powiedz_0.2.orig/def_pars.c	Sat Mar 13 10:38:15 1999
+++ powiedz_0.2/def_pars.c	Sat Jun 23 23:11:52 2001
@@ -20,7 +20,7 @@
 
 #ifdef USE_RC_FILE
 
-#define RC_FILE "/.sayrc";
+#define RC_FILE "/.sayrc"
 
 /* Based on an idea by 
 
@@ -48,15 +48,16 @@
       {
        char string[256];
        int i;
+	   int value;
        union
         {
-         klatt_t def_pars;
+         klatt_frame_t def_pars;
          long array[NPAR];
         }
        pars;
        for (i = 0; i < NPAR; ++i)
         {
-         fgets(string, sizeof(string), infp);
+         fgets(string, sizeof(string), f);
          value = atoi(string);
          pars.array[i] = value;
         }
Only in powiedz_0.2: def_pars.o
Only in powiedz_0.2: dits.o
Only in powiedz_0.2: elements.o
Only in powiedz_0.2: getarg.o
Only in powiedz_0.2: holmes.o
Only in powiedz_0.2: hplay.o
Only in powiedz_0.2: l2u.o
Only in powiedz_0.2: newstring.o
Only in powiedz_0.2: nsynth.o
Only in powiedz_0.2: palput.o
Only in powiedz_0.2: phtoelm.o
Only in powiedz_0.2: polytran.o
Only in powiedz_0.2: powiedz
Only in powiedz_0.2: say.o
